Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.10-slim
+
+ARG api_port
+ENV port ${api_port}
+
+ARG root_path
+ENV path ${root_path}
+
+WORKDIR /root
+COPY ./requirements.txt /root/requirements.txt
+
+# required to install github.com/TheRealNoob/python-logging-loki.git
+RUN apt update && \
+    apt install git curl -y && \
+    apt clean autoclean && \
+    apt autoremove --yes && \
+    rm -rf /var/lib/{apt,dpkg,cache,log}
+
+RUN pip install --no-cache-dir -r /root/requirements.txt
+
+COPY . /root/
+
+CMD uvicorn api.app:app --proxy-headers --host 0.0.0.0 --port ${port} --root-path ${path}
```

Comment:
Is not using the [] syntax poor practice? I can't seem to get env vars to insert into that setup.

Reply:
I could be wrong on this, but I think I had it set up the way you're talking about.  I think this commit broke it.
https://github.com/TheRealNoob/Bot-Detector-Core-Files/commit/7ba82a77f8681bc3468c3d19c6dabd09ecde78f0

Reply:
fixed with https://github.com/TheRealNoob/Bot-Detector-Core-Files/commit/52556b4a751bc3db94a5510b749c65bc2a3edecd